### PR TITLE
Addition of an Issue yml

### PR DESCRIPTION
**Pull request by AbhijithGanesh**
This attempts to solve issue #38. This contains issue.yml. Once approved, I shall make templates for Ideas, bugs and Q and A

### CODE REVIEW

**Comment by kaiwalyakoparkar** [comment]
@AbhijithGanesh could you please enable issues on your fork so that I can test it?

**Comment by AbhijithGanesh** [comment]
Yes

File: .github/ISSUE_TEMPLATE/issue.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
+name: Issue Template❌
```
Comment (kaiwalyakoparkar) [inline .github/ISSUE_TEMPLATE/issue.yml:1]: Replace the ❌ with some generic emoji which matches the name of the issue

**Review by kaiwalyakoparkar — CHANGES_REQUESTED** [review]

**Comment by verma-kunal** [comment]
@patchandpray , I am actually not that much familiar with `yaml` right now (would have to learn) !! 
but i can definitely give a **`lgtm`** after you and @kaiwalyakoparkar would have reviewed this 🙂

**Comment by kaiwalyakoparkar** [comment]
> @patchandpray , I am actually not that much familiar with `yaml` right now (would have to learn) !!
> but i can give a **`lgtm`** after you and @kaiwalyakoparkar would have reviewed this

Yeah, it's good but I suggested for some changes in emoji

File: .github/ISSUE_TEMPLATE/issue.yml (hunk 1 above)
Comment (patchandpray) [reply .github/ISSUE_TEMPLATE/issue.yml:1]: Which emoji did you have in mind. The X does symbolise something beeing an issue so IMO it can be a good fit

**Review by patchandpray — APPROVED** [review]

**Comment by patchandpray** [comment]
> @patchandpray , I am actually not that much familiar with `yaml` right now (would have to learn) !!
> but i can give a **`lgtm`** after you and @kaiwalyakoparkar would have reviewed this

That is fine, YAML stands for Yet Another Markup Language and is a very human friendly (well readable) yet still powerful (supports macros) configuration language. https://circleci.com/blog/what-is-yaml-a-beginner-s-guide/

Kubernetes resources are often defined with YAML as well. (although json is also supported as a configuration language)

**Comment by verma-kunal** [comment]
> > @patchandpray , I am actually not that much familiar with `yaml` right now (would have to learn) !!
> > but i can give a **`lgtm`** after you and @kaiwalyakoparkar would have reviewed this
> 
> That is fine, YAML stands for Yet Another Markup Language and is a very human friendly (well readable) yet still powerful (supports macros) configuration language. https://circleci.com/blog/what-is-yaml-a-beginner-s-guide/
> 
> Kubernetes resources are often defined with YAML as well. (although json is also supported as a configuration language)

thanks!! I am learning this right now, as there is a lot of use of YAML these days, from what i have observed. in DevOps & on GitHub as well

File: .github/ISSUE_TEMPLATE/issue.yml (hunk 1 above)
Comment (verma-kunal) [reply .github/ISSUE_TEMPLATE/issue.yml:1]: i was thinking, do we need to put emojis in all of these!! everything is clear from the name itself, right?

Comment (kaiwalyakoparkar) [reply .github/ISSUE_TEMPLATE/issue.yml:1]: Yes, I insisted him the same :)

**Review by kaiwalyakoparkar — APPROVED** [review]
lgtm

**Pull request merged**